charts/datasaur/templates/start/presidio-analyzer-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
```

Comment:
please add minimum version too. appVersion: ">=6.x.x" e.g. `appVersion: ">=6.69.0"` if it is only applicable to 6.69 onward. Change the version accrodingly.
Also please build the package, e.g. https://github.com/datasaur-ai/helm-charts/pull/46/files#diff-08c65c1b5ef8384ef0200558777c52f4110afcbbacc880ac5375d555ca97ee3eL2

https://github.com/datasaur-ai/helm-charts/pull/46/files#diff-bdedac43eec63509189c5fe23d34c91e1eabd91f9154d9785ec3873f88fa23c6L24

Reply:
updated kak @tonoman3g